src/Frontend/src/app/features/landing/components/landing-page/landing-page.component.html

```
@@ -1,559 +1,691 @@
+                      <span
+                        class="text-[10px] px-1.5 py-0.5 rounded bg-emerald-500/20 text-emerald-400 font-normal"
+                        >HIGH</span
+                      >
+                    </div>
+                  </div>
+
+                  <!-- Typewriter Insight -->
+                  <div
+                    class="font-mono text-sm text-foreground leading-relaxed relative animate-in fade-in slide-in-from-bottom-2 duration-700 delay-1000 fill-mode-both"
+                  >
+                    <span class="text-primary">>></span>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)